sdk/cmake/msvc.cmake

```diff
@@ -279,6 +283,12 @@ function(set_module_type_toolchain MODULE TYPE)
             # Mark .rsrc section as non-disposable non-pageable, as bugcheck code needs to access it
             target_link_options(${MODULE} PRIVATE /SECTION:.rsrc,!DP)
         endif()
+
+        # Clang may insert the call during some optimizations,
+        # but i386 ntoskrnl doesn't export memcmp (before NT 6.2)
+        if(CMAKE_C_COMPILER_ID STREQUAL "Clang" AND ARCH STREQUAL "i386")
+            target_compile_options(${MODULE} PRIVATE -fno-builtin-memcmp)
+        endif()
```

Comment:
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/CMakeLists.txt;h=6849bcf009987f137755d9a2fe26df69374d6ac0;hb=HEAD#l33

Both are fine, but we should be consistent.